### PR TITLE
enable support for ppc64le architecture

### LINKED ISSUE #1493: Add support for ppc64le (power architecture)
**Describe the solution you'd like**

- Get Velero to work on power machines, by adding support for ppc64le.

**Anything else you would like to add:**

- I did build Velero binary and Velero image from source on power architecture and it worked as expected. 

**Environment:**

- Velero version (use `velero version`): v0.11.0
- Kubernetes version (use `kubectl version`): v1.12.4
- Kubernetes installer & version: ICP 3.1.2
- Cloud provider or hardware configuration:
- OS (e.g. from `/etc/os-release`): Ubuntu 16.04

### PR DESCRIPTION
**Pull request by Prajyot-Parab**
Signed-off-by: Prajyot Parab <Prajyot.Parab@ibm.com>

Fixes #1493

### CODE REVIEW

**Comment by skriss** [comment]
@Prajyot-Parab are you intending for us to publish `ppc64le` docker images? It looks like this PR as-is will publish the `ppc64le` binaries for releases on GitHub, but won't publish the docker images, just adds support for building them -- this is fine with me, just checking on what your intention is. 

If you are looking for us to publish images, we might want to consider publishing a multi-arch image using fat manifests, but we can discuss further.

**Comment by Prajyot-Parab** [comment]
> @Prajyot-Parab are you intending for us to publish `ppc64le` docker images? It looks like this PR as-is will publish the `ppc64le` binaries for releases on GitHub, but won't publish the docker images, just adds support for building them -- this is fine with me, just checking on what your intention is.
> 
> If you are looking for us to publish images, we might want to consider publishing a multi-arch image using fat manifests, but we can discuss further.

@skriss getting velero images published for the ppc64le arch would be really helpful. Let me know your thoughts on the same.

**Comment by clnperez** [comment]
We'd love to have the fat manifest support and can help with that if needed. Maybe in a separate PR?

**Comment by jwcroppe** [comment]
@skriss Following up from @clnperez's and @Prajyot-Parab's previous comments - what do you see as next steps?  Merge this and then have a separate PR for handling the Docker images and fat manifest, or is there another route we should take?

**Comment by jwcroppe** [comment]
@skriss Whenever you get a moment, just a quick follow up regarding your thoughts on the above proposal?  Thanks in advance!

**Comment by skriss** [comment]
> what do you see as next steps? Merge this and then have a separate PR for handling the Docker images and fat manifest, or is there another route we should take?

That works for me. I'll take another look through this one as-is shortly.

File: .goreleaser.yml [hunk 1]
```diff
@@ -25,6 +25,7 @@ builds:
       - amd64
       - arm
       - arm64
+      - ppc64le
```
Comment (skriss) [inline .goreleaser.yml]: I believe we only want to build `ppc64le` for `linux` - is that correct? If so, please add entries to the `ignore` list just below to exclude the `ppc64le/darwin` and `ppc64le/windows` combinations from the build.

File: Dockerfile-fsfreeze-pause-ppc64le [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# Copyright 2018, 2019 the Velero contributors.
```
Comment (skriss) [inline Dockerfile-fsfreeze-pause-ppc64le]: This can be just 2019 rather than 2018, 2019.

File: Dockerfile-velero-ppc64le [hunk 3]
```diff
@@ -0,0 +1,32 @@
+# Copyright 2017, 2019 the Velero contributors.
```
Comment (skriss) [inline Dockerfile-velero-ppc64le]: This can be just 2019 rather than 2017, 2019.

File: Dockerfile-velero-restic-restore-helper-ppc64le [hunk 4]
```diff
@@ -0,0 +1,23 @@
+# Copyright 2018, 2019 the Velero contributors.
```
Comment (skriss) [inline Dockerfile-velero-restic-restore-helper-ppc64le]: This can be just 2019 rather than 2018, 2019.

File: .goreleaser.yml (hunk 1 above)
Comment (Prajyot-Parab) [reply .goreleaser.yml]: entries to ignore list added

File: Dockerfile-fsfreeze-pause-ppc64le (hunk 2 above)
Comment (Prajyot-Parab) [reply Dockerfile-fsfreeze-pause-ppc64le]: fixed

File: Dockerfile-velero-ppc64le (hunk 3 above)
Comment (Prajyot-Parab) [reply Dockerfile-velero-ppc64le]: fixed

File: Dockerfile-velero-restic-restore-helper-ppc64le (hunk 4 above)
Comment (Prajyot-Parab) [reply Dockerfile-velero-restic-restore-helper-ppc64le]: fixed

**Pull request merged**